acceptance/acceptance_test.go

```diff
@@ -186,7 +186,7 @@ func testWithoutSpecificBuilderRequirement(
 			})
 
 			it.After(func() {
-				assert.Nil(os.RemoveAll(tmpDir))
+				//assert.Nil(os.RemoveAll(tmpDir))
```

Comment:
Is this just for debugging? Should we remove the validation altogether?

Reply:
Yes sorry